backend/app/controllers/concerns/api/sparse_fieldset.rb

```diff
@@ -0,0 +1,7 @@
+module API
+  module SparseFieldset
+    def sparse_fieldset
+      (params[:fields]&.to_unsafe_h || {}).transform_values { |v| v.split(",") }
```

Comment:
surprisingly rubocop does not complain when using `""` instead of `''` so all good :)

Reply:
We are not using `''`. Just use `""` everywhere. For this project, we are trying to use https://github.com/testdouble/standard

Reply:
Aha, perfect, thanks @tsubik , will check it ;)